premarum-backend/PreEnrollmentMgmt.Application/Repositories/PreEnrollmentRepository.cs

```diff
@@ -23,6 +23,14 @@ public async Task<IEnumerable<PreEnrollment>> GetByStudentIdComplete(int student
                 .ToListAsync();
     }
 
+    public async Task<IEnumerable<PreEnrollment>> GetByStudentIdPartial(int studentId)
```

Comment:
We can delete the other function. The getbystudentid complete since I think it is not being used